docs/content/docs/recipes/teams.md

```diff
@@ -63,21 +63,21 @@ This tutorial assumes that you are familiar with the [Fluid Framework Overview](
 // TODO 6: Define get container function.
 ```
 
-1. Replace `TODO 1:` with the following code. Note that we are exporting the constant because we will be using it to append to the `contentUrl` in the Microsoft Teams settings and parsing the container ID in the content page. It's a common pattern to store important query parameter keys as constants, rather than typing the raw string each time. This is also a good place to define the DDS keys.
+1. Replace `TODO 1:` with the following code. Note that the constant is being exported because it will be used to append to the `contentUrl` in the Microsoft Teams settings and later for parsing the container ID in the content page. It's a common pattern to store important query parameter keys as constants, rather than typing the raw string each time. This is also a good place to define the DDS keys.
 
 ```ts
 export const containerIdQueryParamKey = "containerId";
 ```
 
-1. Replace `TODO: 2` with the following code. Before the client can create any containers, it needs a `containerSchema` that will define, by name, the data objects used in this application. Think of the the `containerSchema` as defining the data structures for how we want our information to be stored. For demonstration purpose, we have a SharedMap as the `initialObjects`, feel free to replace the SharedMap with any other DDSes or data objects. Note also that `map` is the ID of the `SharedMap` object and it must be unique within the container as with any other DDSes.
+1. Replace `TODO: 2` with the following code. Before the client can create any containers, it needs a `containerSchema` that will define, by name, the data objects used in this application. Think of the the `containerSchema` as defining the data structures for how you want your information to be stored. For demonstration purpose, we have a SharedMap as the `initialObjects`, feel free to replace the SharedMap with any other DDSes or data objects. Note also that `map` is the ID of the `SharedMap` object and it must be unique within the container as with any other DDSes.
```

Comment:
For all of these, consider moving the "Replace todo with the following code" to the end. You're telling them to do something, but then they have to read another paragraph before they see the code they're supposed to copy/paste.

Reply:
make sense, moved to the end.